settings.gradle

```
@@ -178,6 +179,7 @@ project(':ballerina-rt').projectDir = file('bvm/ballerina-rt')
 project(':ballerina-treegen').projectDir = file('compiler/ballerina-treegen')
 project(':ballerina-test-utils').projectDir = file('tests/ballerina-test-utils')
 project(':ballerina-io-internal').projectDir = file('misc/io-internal')
+project(':json-mapper').projectDir = file('misc/json-to-record-converter')
```

Wouldn't it be more clearer if we use the directory name as the project name as follows?

```suggestion
project(':json-to-record-converter').projectDir = file('misc/json-to-record-converter')
```

---

Since, there is already a project with the same name, and json-mapper has to be imported there, it causes conflict. Instead if we want we can rename the directory name here.

---

then +1 to keep it as it is